app/visualisation_methods/interaction/_modal.html.haml

```diff
@@ -116,6 +122,7 @@
       query_params += "&param_y="+y_axis;
       query_params += "&output="+outputParam;
       query_params += "&chart_id="+interaction_chart_counter;
+      query_params += "&stand_alone=false";
```

Comment:
@khalska to zadanie: https://jira.plgrid.pl/jira/browse/SCAL-1097
pójdzie na @Andriu01 (tylko teraz nie ma konta w plgridzie...)